mercury/views/measurement.py

```diff
@@ -12,12 +12,42 @@ def build_error(str):
     return json.dumps({"error": str})
 
 
+def add_measurement(request, event):
+    json_data = request.data
+    if isinstance(json_data, str):
```

Comment:
What if json_data is not str?

Reply:
It will be a JSON dict. The only case that the json_data is a string is that it is transferred through radio script since the native request module can only accept the string.

Reply:
Got it. Later it would be great to add assert for the input type (i.e either a JSON dict or a stringified json, otherwise fail loudly). But looks good to me at this point.